update.php

```diff
@@ -1175,3 +1179,18 @@ function update_1505()
 
 	return DBA::delete('config', $conditions) ? Update::SUCCESS : Update::FAILED;
 }
+
+function update_1508()
+{
+	$config = DBA::selectToArray('config');
+
+	$newConfig = DI::config()->transactional();
+
+	foreach ($config as $entry) {
+		$newConfig->set($entry['cat'], $entry['k'], $entry['v']);
+	}
+
+	$newConfig->save();
+
+	DBA::e("DELETE FROM `config`");
```

Comment:
I expect `TRUNCATE` to be more efficient than a condition-less `DELETE`.
```suggestion
	DBA::e("TRUNCATE TABLE `config`");
```